build-service.py

```diff
@@ -4,8 +4,9 @@
 KaaS Repo Build Script
 
 Usage:
-    build-service [-v | -vv] <repo-url> [--branch=<branch>]
-                  [--deploy-conf=<path/to/deploy.yml>] [--service-conf=<path/to/service.yml>]
+    build-service [-v | -vv] <repo-url> [--branch=<branch>] [--restart=<deployment_name>]
+                  [--delete=<deployment_name>] [--deploy-conf=<path/to/deploy.yml>]
+                  [--service-conf=<path/to/service.yml>]
     build-service [-v | -vv] --daemon [--port=<port>]
```

Comment:
Does restart / delete need to go through the docker build process? If not, they could be separate commands / endpoints and not need to build the repo each time, e.g.

```
Usage:
    build-service <repo-url> [--branch=<branch>] [-v | -vv]
                  [--deploy-conf=<path/to/deploy.yml>]
                  [--service-conf=<path/to/service.yml>]
    build-service --restart=<deployment_name>
    build-service --delete=<deployment_name>
    build-service --daemon [--port=<port>] [-v | -vv]
```

Reply:
![image](https://user-images.githubusercontent.com/53018369/162521052-5adf815b-8d02-4cd0-aff0-bb23362c5709.png)
restart and delete do not go through the build process every time